source/main/physics/BeamForcesEuler.cpp

```diff
@@ -341,41 +339,33 @@ void Actor::CalcWheels(bool doUpdate, int num_steps)
                 dbrake = ar_brake_force * abs(ar_hydro_dir_state);
             }
 
-            if ((abrake != 0.0 || dbrake != 0.0 || hbrake != 0.0) && m_num_braked_wheels != 0 )
+            if (abrake != 0.0 || dbrake != 0.0 || hbrake != 0.0)
             {
-                float brake_coef = 1.0f;
-                float antilock_coef = 1.0f;
+                float adbrake = abrake + dbrake; 

```

Comment:
Abrakadabraka FTW 😆

Reply:
If `abrake` and `dbrake` come together you get `adbrake`. Seemed logical. :smile: